ast/format_test.go

```diff
@@ -0,0 +1,222 @@
+package ast_test
+
+import (
+	"testing"
+
+	"github.com/influxdata/flux/ast"
+	"github.com/influxdata/flux/parser"
+	"github.com/pkg/errors"
+)
+
+var skip = map[string]string{
+	"array_expr":           "without pars -> bad syntax, with pars formatting removes them",
+	"conditional":          "how is a conditional expression defined in spec?",
+	"duration_multiple":    "how are multiple duration values meant to be used?",
+	"nil_value_as_default": "cannot distinguish nil values from default values in function definition",
+	"multi_var_decl":       "how is a variable declaration with multiple declarations represented?",
```

Comment:
Each variable declaration should be on its own line. Currently the parser has no syntax for reading multiple declarations into a single VariableDeclaration node so we can ignore that case for now.